src/components/layout/users/user-info-dialog.tsx

```diff
@@ -0,0 +1,246 @@
+    if (!user) return;
+
+    setLoading(true);
+    try {
+      const userProjectIds = user.projectIds;
+      const projects: Project[] = [];
+
+      if (userProjectIds) {
+        for (let i = 0; i < userProjectIds.length; i++) {
+          const project = await projectsApi.findProject({ projectId: userProjectIds[i] });
+          projects.push(project);
+        }
+
+        setUserProjects(projects);
+      }
+    } catch (error) {
+      console.error(t("errorHandling.errorListingUserProjects"), error);
+    }
+    setLoading(false);
+  };
+
+  useEffect(() => {
+    if (!user) return;
+
+    setName(user.firstName + " " + user.lastName);
+    setOrganisation(user.company || "");
+
+    // Fetch user projects
+    getUserProjects();
+  }, [user]);
+
+  /**
+   * Handles user info change
+   * 
+   * TODO: remove if not needed
+   * 
+   * @param field field id
+   */
+  const handleUserInfoChange = (field: string) => (event: React.ChangeEvent<HTMLInputElement>) => {
+    const value = event.target.value;
+    if (field === "name") setName(value);
+    else if (field === "organisation") setOrganisation(value);
+    else if (field === "role") setRole(value);
+  };
+
+  /**
+   * Renders user info section
+   */
+  const renderUserInfoSection = () => (
+    <div style={{ backgroundColor: "rgba(0, 0, 0, 0.1)" }}>
+      <Grid container spacing={1} padding={2}>
+        <Grid item xs={6}>
+          <TextField
+            fullWidth
+            disabled
+            id="name"
+            name="name"
+            label={t("userInfoDialog.name")}
+            placeholder={t("userInfoDialog.name")}
+            variant="outlined"
+            value={name}
+            onChange={handleUserInfoChange("name")}
+          />
+        </Grid>
+        <Grid item xs={6}>
+          <TextField
+            fullWidth
+            disabled
+            id="role"
+            name="role"
+            label={t("userInfoDialog.role")}
+            placeholder={t("userInfoDialog.role")}
+            variant="outlined"
+            value={role}
+            onChange={handleUserInfoChange("role")}
+          />
+        </Grid>
+        <Grid item xs={4}>
+          <TextField
+            fullWidth
+            disabled
+            id="organisation"
+            name="organisation"
+            label={t("userInfoDialog.organisation")}
+            placeholder={t("userInfoDialog.organisation")}
+            variant="outlined"
+            value={organisation}
+            onChange={handleUserInfoChange("organisation")}
+          />
+        </Grid>
+        <Grid item xs={4}>
+          <TextField
+            fullWidth
+            disabled
+            id="lastLogin"
+            name="lastLogin"
+            label={t("userInfoDialog.lastLogin")}
+            placeholder={t("userInfoDialog.lastLogin")}
+            variant="outlined"
+            value={user?.lastLoggedIn?.toLocaleString() || ""}
+            onChange={() => { }}
```

Comment:
Could this be removed? Seems unlikely we would need to change the last logged in date here